[CUBRIDMAN-299] Add descriptions for the missing content in the 'Manual Structure' section. (#636)
CUBRID/cubrid-manual: en/toc.rst

*   :doc:`sql/index`: This chapter describes SQL syntaxes such as data types, functions and operators, data retrieval or table manipulation. The chapter also provides SQL syntaxes used for indexes, triggers, partitioning, serial and user information changes, etc.

*   :doc:`pl/index`: This chapter describes the procedural language extension provided by CUBRID for using stored procedures and stored functions. Stored procedures and functions help handle complex business logic that is difficult to implement with SQL alone and simplify data manipulation tasks.

*   :doc:`admin/index`: This chapter provides instructions on how to create, drop, back up, restore and migrate a database, configuring globalization, and executing CUBRID HA. Also it includes instructions on how to use the  **cubrid** utility, which starts and stops the server, broker, and CUBRID Manager server, etc. Also, this chapter provides instructions on setting system parameters that may influence the performance. It provides information on how to use the configuration file for the server and broker, and describes the meaning of each parameter.

> *   :doc:`ha`: This is the application of the High Availability (HA) feature in CUBRID. It keeps the databases synchronized across multiple servers at all times, and if a failure occurs on the active server, another server automatically takes over the service.

Reviewer: ```suggestion
*   :doc:`ha`: This chapter explains how the High Availability (HA) feature is applied in CUBRID. It keeps databases synchronized at all times across multiple servers, and if a failure occurs on the active server, another server automatically takes over the service.
```

Author: 리뷰내용 수정하였습니다.

> *   :doc:`shard`: Sharding is a method of horizontally partitioning and distributing data across physically separate databases for storage and retrieval. CUBRID SHARD is used to distribute and process large volumes of data and high throughput that cannot be handled by a single database instance.

Reviewer: ```suggestion
*   :doc:`shard`: This chapter explains the sharding feature in CUBRID, which horizontally partitions and distributes data across physically separate databases. CUBRID SHARD is used to distribute and process large volumes of data and high throughput workloads that exceed the capacity of a single database instance.
```

Author: 리뷰내용 수정하였습니다.
